go/analyzer/staticcheck/cmd/all_checks_printer/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/bazelbuild/rules_go/go/analyzer/staticcheck/util"
+)
+
+func main() {
+	for _, a := range util.AnalyzerSlice {
```

Comment:
Ah this is where it's used, I see. I still don't know why though, if all the analysers are in that map. It feels weird to have both, can we stick to just one?

Reply:
agree